test/e2e-secondary-network/secondary_network_test.go

```diff
@@ -98,8 +111,13 @@ func (data *testData) createPodForSecondaryNetwork(ns string, pod *testPodInfo)
 			"App": fmt.Sprintf("%s", podApp),
 		})
 
+	var resNum int64
+	resNum = sriovResNum
+	if pod.isSRIOV {
+		resNum = 1
```

Comment:
I'm a bit confused, it seems resNum is set to the number specific to SRIOV in both cases.

Reply:
In the `testSecondaryNetwork` test case it should set to 3 (sriovResNum    = 3), and it is 1 in this test case:
```
apiVersion: v1
kind: Pod
metadata:
 name: sample-pod-1
 labels:
   app: sample-pod-1
 annotations:
   k8s.v1.cni.cncf.io/networks: sriov-net1
spec:
 containers:
 - name: toolbox
   image: antrea/toolbox:latest
   resources:
      requests:
        intel.com/intel_sriov_netdevice: '1'
      limits:
        intel.com/intel_sriov_netdevice: '1'
```

Reply:
It may make sense at this point to just remove the existing `TestSriovNetwork` (which was meant for baremetal) since we never run it, in order to avoid confusion. cc @jianjuns

Reply:
I am fine to remove the baremetal test.